src/common/NetworkCommon.hpp

```diff
@@ -24,6 +24,13 @@ enum class NetworkRequestType {
     Delete,
     Patch,
 };
+const static std::vector<QString> networkRequestTypes{
+    "GET",     //
+    "POST",    //
+    "PUT",     //
+    "DELETE",  //
+    "PATCH"    //
```

Comment:
This would make diffs cleaner in the future if we're ever to add anything here. It probably won't happen, but it seems cleaner to have that comma at the end imho.
```suggestion
    "PATCH",   //
```